UnityProject/Assets/Scripts/ScriptableObjects/TimedGameEvents/TimedGameEventSO.cs

```diff
@@ -12,9 +12,15 @@ public class TimedGameEventSO : ScriptableObject
 		public Month Month;
 		[Range(1,31)] public int DayOfMonthStart;
 		[Range(1,31)] public int DayOfMonthEnd;
+
 		public string EventName;
-		[NaughtyAttributes.ResizableTextArea] public string EventDesc;
+		[TextArea(10, 20)]
+		public string EventDesc;
+		
 		public Sprite EventIcon;
+
+		//Delete the object when the timed event is not happening
+		public bool deleteWhenNotTime = false;
```

Comment:
I don't understand this, it shouldn't spawn when the even isn't happening

Reply:
It's a mapped thing.

Reply:
Uhmm, then I think the default behaviour should be true